### DISCUSSION TITLE
I can't registered student pack

### DISCUSSION THREAD

**Discussion by afta2160** [community/community#46621] [GitHub Education]
I am a high school student in grade 6. I am not very good at studying and not doing much work and I am poor.  But I love coding.  But I don't have enough money to buy hosting, domain, etc. So I want to create a portfolio for myself.  And this is the school proof from me.  that I am  a student

Here is school website
https://www.swkp.ac.th
(In domain .ac.th (Thai School) domain can be trusted because only the government can see issue it)

**Comment by afta2160** [comment]
![16757689555172552589427456637574](https://user-images.githubusercontent.com/124536046/217231765-52e7297c-3888-4849-8a16-7ce4b61c216d.jpg)

**Comment by SilasRodrigues19** [comment]
Hey @afta2160 

Regarding the Student Developer Pack, I don't know if you can do it with this education level but you can host for free on GitHub and use [GitHub Pages](https://pages.github.com/), [Vercel ](https://vercel.com/dashboard) and [Netlify](https://www.netlify.com/) domains. All for free.

And try signing up for the [Student Developer Pack](https://education.github.com/) too
